From 80fefaeb573bd6072ac02b8bb14a1b87fe74de37 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms@verge.net.au>
Date: Mon, 25 Nov 2013 10:46:34 +0900
Subject: [PATCH 129/179] MEDIUM: Log agent fail, stopped or down as info

In the case where an agent check returns fail, stopped or down,
log this as info when logging the server status along with any
trailing message returned by the agent after fail, stopped or down.

Previously only the trailing message was logged as info and
if omitted no info was logged.

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 src/checks.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/checks.c b/src/checks.c
index 6b7c382..d865c0b 100644
--- a/src/checks.c
+++ b/src/checks.c
@@ -1010,10 +1010,7 @@ static void event_srv_chk_r(struct connection *conn)
 			 */
 			if (end[0] == '\0' || end[0] == ' ' || end[0] == '\t') {
 				status = HCHK_STATUS_L7STS;
-				/* Skip over leading blanks */
-				while (end[0] != '\0' && (end[0] == ' ' || end[0] == '\t'))
-					end++;
-				desc = end;
+				desc = check->bi->data;
 			}
 		}
 
-- 
1.8.1.5

